x-pack/plugins/security_solution/public/app/app.tsx

```diff
@@ -81,6 +86,10 @@ const StartAppComponent: FC<StartAppComponent> = ({
                 <AssistantProvider
                   actionTypeRegistry={actionTypeRegistry}
                   augmentMessageCodeBlocks={augmentMessageCodeBlocks}
+                  defaultAllow={defaultAllow}
+                  defaultAllowReplacement={defaultAllowReplacement}
+                  baseAllow={DEFAULT_ALLOW}
+                  baseAllowReplacement={DEFAULT_ALLOW_REPLACEMENT}
```

Comment:
We're getting beefy here but should clean up once we move these to localstorage inside the context 👍